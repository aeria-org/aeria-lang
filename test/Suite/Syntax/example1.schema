collection User {
  properties {
    first_name      str
    last_name       str
  }
}
