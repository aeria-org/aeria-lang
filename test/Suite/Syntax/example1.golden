(Program {
  collections:
    ((Collection
      { getters: Nil
      , name: (CollectionName "user")
      , properties:
        ((Property { attributes: Nil, name: (PropertyName "first_name"), type_: PString }) :
         (Property { attributes: Nil, name: (PropertyName "last_name"), type_: PString }) : Nil)
      , required: Nil
      , table: Nil
      }) : Nil) })
